profiles/sftp_storage/compose.yaml

```diff
@@ -0,0 +1,8 @@
+version: "3.7"
+
+services:
+  sftp_storage:
+    image: atmoz/sftp
+    ports:
+      - "9922:22"
```

Comment:
Oci-env can be used multiple times with multiple compose.env files, binding to one port will prevent other containers to one port. Make this  a config variable.

Reply:
Do we even need to bind the port on localhost? Can we make the containers talk to each other without it?